Resources/Private/Language/Labels.xml

```diff
@@ -192,7 +192,7 @@
             <label index="config.solrHttps">Use https: (default is "FALSE")</label>
             <label index="config.solrHost">Solr Server Host: (default is "localhost")</label>
             <label index="config.solrPort">Solr Server Port: (default is "8983")</label>
-            <label index="config.solrPath">Solr Server Path: (default is "/solr/")</label>
+            <label index="config.solrPath">Solr Server Path: without API endpoint "/solr" (default is "/")</label>
```

Comment:
I'm not getting this comment. When it is /solr?

Reply:
If your Solr runs on `localhost:8983/some-path/solr` in the past, you had to set `config.solrPath` to `/some-path/solr`.
Since newer versions of Solarium add the `/solr` part automatically (because it is considered to be the API endpoint, but not part of the application's path), you have to omit this part and just set `config.solrPath` to `/some-path`.
I added the comment to (hopefully) make this clear - especially for users who are used to the old way...